src/compiler/scala/tools/nsc/backend/jvm/analysis/ProdConsAnalyzerImpl.scala

```diff
@@ -435,7 +435,7 @@ trait ProdConsAnalyzerImpl {
  * The ASM built-in SourceValue analysis yields an empty producers set for such values. This leads
  * to ambiguities. Example (in Java one can re-assign parameter):
  *
- *   void foo(int a) {
+ *   int foo(int a) {
```

Comment:
Just scouting out around here, in case anyone decides to compile the java in the scaladoc and gets surprised.